cli/tests/unit/webgpu_test.ts

```diff
@@ -0,0 +1,242 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+import { assert, assertEquals } from "./test_util.ts";
+
+let isCI: boolean;
+try {
+  isCI = (Deno.env.get("CI")?.length ?? 0) > 0;
```

Comment:
This is 4th place we define this env var :) we should move it to `test_util.ts` in a follow up. Can you handle that @crowlKats?

Reply:
sure